src/main/java/de/tum/in/www1/artemis/web/rest/FileUploadExerciseResource.java

```diff
@@ -356,7 +332,7 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(fileUploadExerc
 
         public static final String ROOT = "/api";
```

Comment:
It is a bit strange that we have both the string literals in the annotations as well as the nested `Endpoints` class with constants. We might want to decide on one approach, but this does not have to be in this PR. But it might fit well into those refactorings in general.